app/views/hyrax/collections/_form_share.html.erb

```diff
@@ -6,7 +6,7 @@
   </legend>
 
   <div class="alert alert-info hidden" id="save_perm_note">Permissions are
-  <strong>not</strong> saved until the &quot;Save&quot; button is pressed at the bottom of the page.</div>
+  <strong>not</strong> saved until the &quot;Update Collection&quot; button is pressed at the bottom of the page.</div>
```

Comment:
This would be a fairly simple change, so I could add it in... the only reason I didn't was because redoing the configuration changes were a bit more of a change and I wasn't sure how much time I had to spend on this issue. It seemed to make sense to do the translations all at once.  I assume this is a change you want made for all three partials?